cli/defenseclaw/tui/panels/setup.py

```
@@ -3942,6 +4515,21 @@ def missing_required_fields(wizard: SetupWizard | int, fields: Sequence[WizardFo
         if action in {"add", "remove"} and not wizard_field_value(fields, "Directory"):
             missing.append("Directory")
+    if wizard == SetupWizard.REDACTION:
+        action = wizard_field_value(fields, "Action") or "status"
+        if action in {"apply-all", "apply-defaults"} and not wizard_field_value(fields, "Profile"):
+            missing.append("Profile")
+        if action in {"profile-show", "profile-set", "profile-remove"} and not wizard_field_value(
+            fields, "Custom Profile Name"
+        ):
+            missing.append("Custom Profile Name")
+        if action.startswith("destination-") or action.startswith("route-"):
+            if not wizard_field_value(fields, "Destination"):
```

Ruff (0.16.1)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)